reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientFormEncoder.java

```diff
@@ -80,7 +78,7 @@
 		this.newCharset = charset;
 		this.request = request;
 		this.cleanOnTerminate = true;
-		this.progressFlux = DirectProcessor.create();
+		this.progressSink = Sinks.many().multicast().directBestEffort();
```

Comment:
just as a reminder: if there are multiple subscribers with different demand, the slowest subscribers will not see pushed values (when they're at request 0) but they won't be terminated either, contrary to `DirectProcessor`. if NONE of the subscribers have enough request to process the value, however, the sink terminates with an overflow exception

Reply:
> just as a reminder: if there are multiple subscribers with different demand, the slowest subscribers will not see pushed values (when they're at request 0) but they won't be terminated either, contrary to `DirectProcessor`.

Reactor Netty is the subscriber here, no other subscribers.

> if NONE of the subscribers have enough request to process the value, however, the sink terminates with an overflow exception

Reactor Netty may stop requesting if the channel is not writable.

Reply:
if there is a single subscriber, wouldn't `unicast()` fit better? note there is two flavors: onBackpressureBuffer and onBackpressureError

Reply:
ok I'll change that to `unicast()`

about back pressure strategy: till now we have `progressSink.asFlux().onBackpressureLatest()`?

Reply:
please check